content/blog/2018-12-11-treestartr.md

```diff
@@ -0,0 +1,101 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+And I hope I can answer this question adequately. I picked ROpenSci because it was explicitly R (so the editor and reviewers would know best practices) and the guidelines and how review would proceed were very clear. And for a totally new experience of in-depth code review, being able to see where the effort was collaborative was helpful. I looked at [other journals](https://joss.theoj.org/about#author_guidelines), but I liked that ROpenSci had biologists as editors. Because the handling editor ([Scott Chamberlain](https://scottchamberlain.info/)) is a biologist, he had the ability to work with me to find reviewers ([David Bapst](https://geogeo.tamu.edu/people/faculty/bapstdavidwilliam) and [Rachel Warnock](https://www.bsse.ethz.ch/cevo/the-group/people/person-detail.html?persid=215787)) who are serious experts in both R and the particular type of phylogenetics that I do. I chose Dave & Rachel because I know I can collaborate with them, and they will be constructive and kind. But they will also really try to break the package and dig in to the meat of what software functions are required to support the science we do, and tell me honestly if my package is providing those functions. [You can see](https://github.com/ropensci/onboarding/issues/239) they weren't easy on me!
+
+ROpenSci also has a collaboration with [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/author-guidelines). I plan to send a short software note there. It's very nice to have the flexibility to send the paper to a traditional outlet like MEE, or to send it to [JOSS](https://joss.theoj.org/), or to do nothing at all, if I didn't want to. 
+
+### tl;dr
+
+Come on in, biologists. The water's fine. 
+
+## Future
+
+I'll likely add in more utility functions for parsing biological data for use with divergence time estimation. Particularly, I often work with [RevBayes](https://revbayes.github.io/), which is a sandbox-like framework for Bayesian analysis of phylogeny and comparative methods. Our analyses integrate across fossil taxa from databases, molecular data from the literature and stratigraphic ranges. Each data source has special and unique quirks, and I'm always looking for ways to lower the bar for undergraduate researchers to be able to access these sorts of complex modeling frameworks. I have a [Contributing](https://github.com/ropensci/treeStartR/blob/master/.github/CONTRIBUTING.md) guide here; please feel free to make a pull request or file an issue if there's something you'd like to see added.
```

Comment:
Can you change the order of these phrases? It sounds like you want pull requests to your contributing guide
" I have a Contributing guide here; please feel free to make a pull request or file an issue if there’s something you’d like to see added."